Source/DSPTool/StubHost.cpp

```diff
@@ -0,0 +1,66 @@
+// Copyright 2014 Dolphin Emulator Project
```

Comment:
2017

Reply:
The comment below shouldn't reference the tests also

Reply:
Somewhat unrelated, but should I replace the Common/GL/GLInterfaceBase.h include with a forward declaration for cInterfaceBase instead?

Reply:
If [InitInterface](https://github.com/dolphin-emu/dolphin/blob/ccfc0816974bde303ac62bfad019a41ff6fb13f0/Source/Core/Common/GL/GLUtil.cpp#L16) for GLUtil will never be called during DSPTool's lifecycle

```cpp
std::unique_ptr<cInterfaceBase> HostGL_CreateGLInterface()
{
  return nullptr;
}
```

shouldn't even be necessary

Reply:
Yeah it shouldn't ever be called. I'll remove it then.